charts/indexer/templates/secrets.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
Make it optional like:
- https://github.com/InstrumentalFi/helm-charts/blob/f2cf83a1db1384a53b3f6b5b4b0cab2187796087/charts/autocompounder/templates/secret.yaml#L1
- https://github.com/InstrumentalFi/helm-charts/blob/f2cf83a1db1384a53b3f6b5b4b0cab2187796087/charts/autocompounder/values.yaml#L14-L17
- so when deploying from gitops, we can disable secret creation and provide separatly a `SealedSecret` object